docs/reference/reana-yaml/index.md

```diff
@@ -64,6 +64,18 @@ The **inputs** property is composed of:
 | directories | *list*       | *optional* | Lists all the input directories to the workflow. Will be seeded to the workspace before running.                                 |
 | files       | *list*       | *optional* | Lists all the input files to the workflow.  Will be seeded to the workspace before running.                                      |
 | parameters  | *dictionary* | *optional* | Specifies all the input parameters to the workflow. It is a dictionary of parameter names and their values expressed as strings. |
+| options     | *dictionary* | *optional* | Specifies operational options for each workflow engine, see below.                                                               |
+
+The **inputs.options** property describes operational options that can be used for the different workflow engines. The available options are:
+
+| Property | Type     | Mandatory?  | Workflow engine | Description                            |
+| -------- | ----     | ----------  | ----------------|                                        |
+| CACHE    | *string* | *optional*  | Serial          | Whether the workflow engine should cache the results of each step for faster execution of subsequent workflow runs. Disabled by default. Can be `on` or `off`. |
+| FROM     | *string* | *optional*  | Serial          | Allows partial execution of a workflow starting from the beginning of a desired step. The value is the name of the desired starting step.  Note that the `FROM` option be combined with the `TARGET` option. |
+| TARGET   | *string* | *optional*  | Serial, CWL     | Allows partial execution of a workflow until the end of a desired step. The value is the name of the desired target step. Note that the `TARGET` option can be combined with the `FROM` option. |
+| toplevel | *string* | *optional*  | Yadage          | Yadage `toplevel` argument. It represents the working directory or remote repository where the workflow should be pulled from. [More info](https://yadage.readthedocs.io/en/latest/definingworkflows.html?highlight=toplevel#using-json-references). It supports GitHub as remote repository `github:<username/repo[@branch]>[:subpath]` |
+| initdir  | *string* | *optional*  | Yadage          | Yadage `initdir` argument. It represents the initial directory for workflows running locally. |
+| initfiles  | *list* | *optional*  | Yadage          | Yadage `initfiles` argument. A list of YAML files that sets the inital data. |
```

Comment:
One more thing: `s/inital/initial/g`.

Reply:
BTW the phrase "sets initial data" may not be clean to users, as it does not contain data files properly speaking, only references, as it were. Better to expand the phrase a bit, such as "...that pass initial parameters as the workflow inputs" or something of the kind. Ideally with a reference to Yadage docs.